root/io/filemerger/CMakeLists.txt

```diff
@@ -1,7 +1,359 @@
+                hsimple.root)
+endif()
+
+if(${compression_default} STREQUAL "zlib")
+   if(ZLIB_CF)
+      ROOTTEST_ADD_TEST(simple-default-compr-level9
+                  PRECMD ${ROOT_hadd_CMD} -f9 hsimple9.root hsimple.root
+                  COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple9.root\",25000,9,441354,5)"
+                  DEPENDS hsimple-filemergerfile)
+
+      ROOTTEST_ADD_TEST(simple-zlib-compr-level1
+                  PRECMD ${ROOT_hadd_CMD} -f101 hsimple101.root hsimple.root
+                  COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple101.root\",25000,101,442867,5)"
+                  DEPENDS hsimple-filemergerfile)
+
+      ROOTTEST_ADD_TEST(simple-zlib-compr-level6
+                  PRECMD ${ROOT_hadd_CMD} -f106 hsimple106.root hsimple.root
+                  COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple106.root\",25000,106,441460,5)"
+                  DEPENDS hsimple-filemergerfile)
+
+      ROOTTEST_ADD_TEST(simple-zlib-compr-level9
+                  PRECMD ${ROOT_hadd_CMD} -f109 hsimple109.root hsimple.root
+                  COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple109.root\",25000,109,441364,5)"
+                  DEPENDS hsimple-filemergerfile)
+
+      ROOTTEST_ADD_TEST(simple-default-compr-level9-datageneration
+                  PRECMD  ${ROOT_hadd_CMD} -f9 hsimple9x2.root hsimple.root
+                  COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple9x2.root\",25000,9,441364,5)"
+                  POSTCMD ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/hsimple9x2.root ${CMAKE_CURRENT_BINARY_DIR}/hsimple92.root
+                  DEPENDS hsimple-filemergerfile)
+
+      ROOTTEST_ADD_TEST(simple-zlib-compr-level9-datageneration
+                  PRECMD  ${ROOT_hadd_CMD} -f109 hsimple109x2.root hsimple.root
+                  COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple109x2.root\",25000,109,441371,5)"
+                  POSTCMD ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/hsimple109x2.root ${CMAKE_CURRENT_BINARY_DIR}/hsimple1092.root
+                  DEPENDS hsimple-filemergerfile)
+
+      ROOTTEST_ADD_TEST(simplex2-default-compr-level9
+                  PRECMD  ${ROOT_hadd_CMD} -f9 -a hsimple9x2.root hsimple.root
+                  COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple9x2.root\",50000,9,870903,10)"
+                  POSTCMD ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/hsimple9x2.root ${CMAKE_CURRENT_BINARY_DIR}/hsimple92.root
+                  DEPENDS simple-default-compr-level9-datageneration)
+
+      ROOTTEST_ADD_TEST(simplex2-zlib-compr-level9
+                  PRECMD  ${ROOT_hadd_CMD} -f109 -a hsimple109x2.root hsimple.root
```

Comment:
This seems to use the same data file (name) as simple-zlib-compr-level9-datageneration

Reply:
I am confused, but here I am using hadd -a option, right?

Reply:
I missed the '-a' :) [which means that we need to bring addition to it either by making the test name more explicit or adding a comment.]  thanks.